sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -64,11 +63,11 @@ typedef struct
   az_span user_agent;
 
   /**
-   * The `method_twin_content_type` is sent in the username to inform IoT Hub what format the device
-   * expects Twin Document and Direct Method payloads to use. If this option is not set, the default
+   * The `twin_content_type` determines the format to send in the username, which will inform IoT
+   * Hub what format the device expects Twin Document to use. If this option is not set, the default
```

Comment:
```suggestion
   * Hub what format the device expects twin document to use. If this option is not set, the default
```

Reply:
is direct methods, telemetry, and c2d also not capitalized?  What is the standard on all these?